pkg/node/manager/manager.go

```diff
@@ -294,11 +299,13 @@ func (m *manager) backgroundSync(ctx context.Context) error {
 			m.mutex.RUnlock()
 			m.Iter(func(nh datapath.NodeHandler) {
 				nh.NodeValidateImplementation(entry.node)
+
 			})
 			entry.mutex.Unlock()
 
 			m.metricDatapathValidations.Inc()
 		}
+		m.healthReporter.OK("All good here! Nothing to see")
```

Comment:
We can drop this code from this PR, I trust that this is all working and will pick up any modules reporting a status in future PRs.

Reply:
Indeed... Thanks Tom!